test/units/modules/test_iptables.py

```diff
@@ -1,1174 +1,1402 @@
-            'source': '192.168.0.1/32',
-            'destination_ports': ['80', '443', '8081:8085'],
-            'jump': 'ACCEPT',
-            'comment': 'this is a comment',
-        })
-        commands_results = [
-            (0, '', ''),
-        ]
-
-        with patch.object(basic.AnsibleModule, 'run_command') as run_command:
-            run_command.side_effect = commands_results
-            with self.assertRaises(AnsibleExitJson) as result:
-                iptables.main()
-                self.assertTrue(result.exception.args[0]['changed'])
-
-        self.assertEqual(run_command.call_count, 1)
-        self.assertEqual(run_command.call_args_list[0][0][0], [
-            '/sbin/iptables',
-            '-t', 'filter',
-            '-C', 'INPUT',
-            '-p', 'tcp',
-            '-s', '192.168.0.1/32',
-            '-j', 'ACCEPT',
-            '-m', 'multiport',
-            '--dports', '80,443,8081:8085',
-            '-i', 'eth0',
-            '-m', 'comment',
-            '--comment', 'this is a comment'
-        ])
-
-    def test_match_set(self):
-        """ Test match_set together with match_set_flags """
-        tests = [
             [
-                {
-                    "chain": "INPUT",
-                    "protocol": "tcp",
-                    "match_set": "admin_hosts",
-                    "match_set_flags": "src",
-                    "destination_port": "22",
-                    "jump": "ACCEPT",
-                    "comment": "this is a comment",
-                },
-                [
-                    "/sbin/iptables", "-t", "filter",
-                    "-C", "INPUT", "-p", "tcp",
-                    "-j", "ACCEPT", "--destination-port", "22",
-                    "-m", "set", "--match-set", "admin_hosts",
-                    "src", "-m", "comment", "--comment", "this is a comment",
-                ],
+                IPTABLES_CMD,
+                "-t",
+                "filter",
+                "-C",
+                "INPUT",
+                "-m",
+                "iprange",
+                "-j",
+                "ACCEPT",
+                "--src-range",
+                "192.168.1.100-192.168.1.199",
             ],
+            id="ip_range_with_src_range",
+        ),
+        pytest.param(
+            {
+                "chain": "INPUT",
+                "src_range": "192.168.1.100-192.168.1.199",
+                "dst_range": "10.0.0.50-10.0.0.100",
+                "jump": "ACCEPT",
+            },
+            [
+                IPTABLES_CMD,
+                "-t",
+                "filter",
+                "-C",
+                "INPUT",
+                "-j",
+                "ACCEPT",
+                "-m",
+                "iprange",
+                "--src-range",
+                "192.168.1.100-192.168.1.199",
+                "--dst-range",
+                "10.0.0.50-10.0.0.100",
+            ],
+            id="ip_range_with_src_range_dst_range",
```

Comment:
same concern as above the test function name already has the `ip_range` mention. so lets dedupe so that the test IDs look like `test_iprange[src-range-and-dst-ranges-combo]` ("with" seems redundant plus it's used inconsistently across the params)